src/stats/hmm.rs

```diff
@@ -199,6 +200,15 @@ pub trait Model<Observation> {
     /// Transition probability between two states `from` and `to`.
     fn transition_prob(&self, from: State, to: State) -> LogProb;
 
+    /// Transition probability between two states `from` and `to` for observation with index
+    /// `_to_idx` (index of `to`).
```

Comment:
This maybe needs some motivating sentences. Or point to a textbook or paper.

Reply:
Done.